### PR TITLE
Document possible getLocale returns values.  

### LINKED ISSUE #5363: `app.getLocale()` inconsistent with chromium documented supported locales
* Electron version: `0.37.8`
* Operating system: Windows

When calling `app.getLocale()` I get the value `"en-GB"`.  Although this is technically correct, chromiums [list of locales](https://developer.chrome.com/webstore/i18n#localeTable) indicates it should be `en_GB`.

This breaks certain locale and translation packages that I was looking at using, although the fix on my end was a simple `.replace('-', '_')` I'm wondering why the inconstency with chromium API when as far as I can tell from the [relevant code](https://github.com/electron/electron/blob/d9db23185cba1bc7cedc3b73a56af3093812a726/atom/browser/api/atom_api_app.cc#L375) we are only calling chromium's built in locale lookup?

### PR DESCRIPTION
**Pull request by MarshallOfSound**
Closes #5363

### CODE REVIEW

File: docs/api/app.md [hunk 1]
```diff
@@ -396,7 +396,7 @@ Overrides the current application's name.
 
 ### `app.getLocale()`
 
-Returns the current application locale.
+Returns the current application locale. Possible return values are documented [here](../faq/app-locale.md)
```
Comment (zcbenz) [inline docs/api/app.md]: I'm not sure if we should put this file under `faq/`, other docs like `chrome-command-line-switches.md` and `environment-variables.md` are put under `api/`.

File: docs/faq/app-locale.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
+Possible Values of `app.getLocale()`
```
Comment (zcbenz) [inline docs/faq/app-locale.md]: Our current style of docs requires the title to be like below:

```markdown
# Possible Values of `app.getLocale()`
```

https://github.com/electron/electron/blob/master/docs/styleguide.md#titles

File: docs/api/app.md (hunk 1 above)
Comment (MarshallOfSound) [inline docs/api/app.md]: Yeah, wasn't sure where it belonged.  I'll move it to `api/` 👍

**Comment by zcbenz** [comment]
👍

**Pull request merged**